package.json

```diff
@@ -1,68 +1,30 @@
 {
-  "name": "structure",
+  "name": "secret-admirer",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "babel": {
-    "presets": [
-      "@babel/preset-env",
-      "@babel/preset-react"
-    ],
-    "plugins": [
-      [
-        "@babel/plugin-proposal-class-properties"
-      ]
-    ]
-  },
   "scripts": {
-    "test": "mocha",
-    "start": "PORT=80 concurrently \"nodemon app\" \"webpack-dev-server --open\"",
-    "create": "webpack",
-    "dev": "nodemon app",
-    "client": "webpack-dev-server --open"
+    "server": "PORT=80 node server.js",
+    "server-dev": "nodemon server.js",
+    "client": "cd client && npm run build",
+    "client-dev": "cd client && npm run client-dev",
+    "dev": "concurrently --kill-others-on-fail \"npm run server-dev\" \"npm run client-dev\"",
+    "prod": "PORT=80 concurrently --kill-others-on-fail  \"npm run client\" \"node server.js\""
```

Comment:
no concurrently, build first and run server. also run server by `npm run server`